scripts/gcp-build-and-push.sh

```diff
@@ -38,6 +61,28 @@ for tag in "$version-commercial" "beta" "$timestamp"; do
     docker push "$image:$tag"
 done
 
+# Install gcrane if not already installed (only if we have regions to replicate to)
+if [ ${#REPLICA_REGIONS[@]} -gt 0 ]; then
+    if ! command -v gcrane &> /dev/null; then
+        echo "Installing gcrane..."
+        go install github.com/google/go-containerregistry/cmd/gcrane@latest
```

Comment:
do we need go dependencies ?  probably on the `.github/workflows/gcp-build-and-push.yml`